book/src/user/lightwalletd.md

```diff
@@ -0,0 +1,248 @@
+
+Contents:
+
+- [Download and build Zebra](#download-and-build-zebra)
+- [Configure zebra for lightwalletd](#configure-zebra-for-lightwalletd)
+  - [RPC section](#rpc-section)
+  - [State section](#state-section)
+  - [Network section](#network-section)
+- [Sync Zebra](#sync-zebra)
+- [Download and build lightwalletd](#download-and-build-lightwalletd)
+- [Sync lightwalled](#sync-lightwalled)
+- [Run tests](#run-tests)
+- [Connect wallet to lightwalletd](#connect-wallet-to-lightwalletd)
+  - [Download and build the cli-wallet](#download-and-build-the-cli-wallet)
+  - [Run the wallet](#run-the-wallet)
+- [Lightwalletd and nginx](#lightwalletd-and-nginx)
+
+## Download and build Zebra
+[#download-and-build-zebra]: #download-and-build-zebra
+
+```console
+git clone https://github.com/ZcashFoundation/zebra
+cd zebra
+cargo build --release
+```
+
+Zebra binary will be at `target/release/zebrad`.
+
+## Configure zebra for lightwalletd
+[#configure-zebra-for-lightwalletd]: #configure-zebra-for-lightwalletd
+
+We need a zebra configuration file. First, we create a file with the default settings:
+
+```console
+./target/release/zebrad generate -o zebrad.toml
+```
+
+This will create a `zebrad.toml` file. Tweak the following options in order to prepare for lightwalletd setup.
+
+### RPC section
+[#rpc-section]: #rpc-section
+
+This change is required for zebra to behave as an RPC endpoint. The standard port for RPC endpoint is `8232`.
+
+```
+[rpc]
+listen_addr = "127.0.0.1:8232"
+```
+
+### State section
+[#state-section]: #state-section
+
+Make sure your cache is in a hard drive with enough space to hold the full blockchain (around 30G at the moment of writing - 2022-05-28).
+
+```
+[state]
+cache_dir = "/lot/of/free/space/for/zebra"
+...
+```
+
+### Network section
+[#network-section]: #network-section
+
+Only change this if you want to sync for the testnet:
+
+```
+[network]
+...
+network = 'Testnet'
+...
+```
+
+## Sync Zebra
+[#sync-zebra]: #sync-zebra
+
+With the configuration in place you can start synchronizing Zebra with the Zcash blockchain. This may take a while depending on your hardware.
+
+```console
+./target/release/zebrad -c zebrad.toml start
+```
+
+Zebra will display information about sync process:
+
+```console
+...
+zebrad::commands::start: estimated progress to chain tip sync_percent=10.783 % 
+...
+```
+
+Until eventually it will get there:
+
+```console
+...
+zebrad::commands::start: finished initial sync to chain tip, using gossiped blocks sync_percent=100.000 % 
+...
+```
+
+You can interrupt the process at any time with `ctrl-c` and Zebra will resume the next time at around the block you were downloading when stopping the process.
+
+When deploying for production infrastructure, the above command can/should be implemented as a server service or similar configuration. 
+
+For implementing zebra as a service please see [here](https://github.com/ZcashFoundation/zebra/blob/v1.0.0-beta.10/zebrad/systemd/zebrad.service).
```

Comment:
We don't want versioned links in long-term documentation, because we'd have to change them every release:

```suggestion
For implementing zebra as a service please see [here](https://github.com/ZcashFoundation/zebra/blob/main/zebrad/systemd/zebrad.service).
```